src/main-process/parse-command-line.js

```diff
@@ -177,7 +193,34 @@ module.exports = function parseCommandLine (processArgs) {
   }
 }
 
-function normalizeDriveLetterName (filePath) {
+const readProjectSettingsSync = (filepath, executedFrom) => {
+  if (!hasAtomProjectFormat(path.basename(filepath))) {
+    throw new Error('File must match format: *.atomproject.{json, cson}')
```

Comment:
Since we have an explicit flag for specifying a project, I don't think we need to actually enforce that the filename matches any particular pattern. Thoughts?

Reply:
I had the same thought while writing this - I'm fine with not enforcing a filename format!